django/db/migrations/operations/base.py

```diff
@@ -1,6 +1,17 @@
+import enum
+
 from django.db import router
 
 
+class OperationCategory(str, enum.Enum):
+    ADDITION = "+"
+    REMOVAL = "-"
+    ALTER = "~"
+    PYTHON = "p"
+    SEPARATE_DATABASE_AND_STATE = "?"
```

Comment:
I'd prefer `MIXED` as suggested by Simon, there may be other custom operations matching this, so it's better to use a more generic name.

Reply:
Aha! Got it! I was confused and thought maybe Simon suggested changing its naming to the mixed case! I reckon now it's ok